face/VatomView.js

```diff
@@ -37,14 +36,41 @@ export default class VatomView {
+    this.createLoader = this.config.loader || function()  {
+      let loader = document.createElement('div');
+      loader.style.cssText = 'position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;';
+      loader.innerHTML = '<svg width="100%" height="100%" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="xMidYMid" class="lds-double-ring"><circle cx="50" cy="50" ng-attr-r="{{config.radius}}" ng-attr-stroke-width="{{config.width}}" ng-attr-stroke="{{config.c1}}" ng-attr-stroke-dasharray="{{config.dasharray}}" fill="none" stroke-linecap="round" r="40" stroke-width="4" stroke="#456caa" stroke-dasharray="62.83185307179586 62.83185307179586" transform="rotate(305.714 50 50)"><animateTransform attributeName="transform" type="rotate" calcMode="linear" values="0 50 50;360 50 50" keyTimes="0;1" dur="1s" begin="0s" repeatCount="indefinite"></animateTransform></circle><circle cx="50" cy="50" ng-attr-r="{{config.radius2}}" ng-attr-stroke-width="{{config.width}}" ng-attr-stroke="{{config.c2}}" ng-attr-stroke-dasharray="{{config.dasharray2}}" ng-attr-stroke-dashoffset="{{config.dashoffset2}}" fill="none" stroke-linecap="round" r="35" stroke-width="4" stroke="#88a2ce" stroke-dasharray="54.97787143782138 54.97787143782138" stroke-dashoffset="54.97787143782138" transform="rotate(-305.714 50 50)"><animateTransform attributeName="transform" type="rotate" calcMode="linear" values="0 50 50;-360 50 50" keyTimes="0;1" dur="1s" begin="0s" repeatCount="indefinite"></animateTransform></circle></svg>';
+      return loader
     }
 
+    this.createErrorView = this.config.errorView || function (bvi, v, err) {
+      
+      let con = document.createElement('div');
+      const rN = 'ActivatedImage';
+      const rs = v.properties.resources.find(r => r.name === rN);
+      const du = bvi.UserManager.encodeAssetProvider(rs.value.value);
```

Comment:
Need to check for the case where `rs` is null (no ActivatedImage found). Maybe just change this line to:

``` js
const du = rs && bvi.UserManager.encodeAssetProvider(rs.value.value)
```

Reply:
af119e2